src/components/controls/TextArea/TextArea.tsx

```diff
@@ -32,6 +32,8 @@ export type TextAreaProps = BaseInputControlProps<HTMLTextAreaElement> & {
     maxRows?: number;
     /** An optional element displayed under the lower right corner of the control and sharing the place with the error container */
     note?: React.ReactNode;
+    /** Indicates that the user cannot change control's value */
+    readOnly?: boolean;
```

Comment:
I suggest you to move this property to [BaseInputControlProps](https://github.com/gravity-ui/uikit/blob/c4a9d97413bb24539270edc54b2eb6e02e2bcb56/src/components/controls/types.ts#L22) type

Reply:
[done](https://github.com/gravity-ui/uikit/pull/1747/commits/55179ed111883e99bea2aef479f9ab07f522ea2f)